fileattrs/sysusers.attr

```diff
@@ -20,3 +20,41 @@
         ::continue::
     end
 }
+
+%__sysusers_recommends() %{lua:
+    if rpm.expand('%[0%{?_use_weak_usergroup_deps}]') == '0' then
```

Comment:
Oh and there's indeed a nicer way to do this (I remember you asking about this but didn't have an answer off-hand):

```
if tonumber(macros._use_weak_usergroup_deps) == 0 then
```

```

Reply:
Funny enough `tonumber` returns `nil` on `nil` so that won't work unless augmented with some handling for that case.

Reply:
Argh, indeed. Maybe we should just add a native macro for rpmExpandNumeric(), it's not like the need to check for config setting is a rare case...